docs/api/robottelo.cli.rst

```diff
@@ -60,6 +60,11 @@ Submodules:
 
 .. automodule:: robottelo.cli.environment
 
+:mod:`robottelo.cli.erratum`
+--------------------------------
```

Comment:
A nitpick maybe, but above 2 lines should have exactly the same length:
```
:mod:`robottelo.cli.erratum`
----------------------------
```
Same for the rest of lines:
```
:mod:`robottelo.cli.job_invocation`
-----------------------------------
```
and so on.